src/libsyntax_ext/assert.rs

```diff
@@ -86,7 +761,7 @@ fn escape_format_string(s: &str) -> String {
 
 #[test]
 fn test_escape_format_string() {
-    assert!(escape_format_string(r"foo{}\") == r"foo{{}}\\");
+    assert!(escape_format_string("foo{}") == "foo{{}}");
```

Comment:
Is this change necessary? The implementation of `escape_format_string` has not changed.